nginx-app/spec/redirector_spec.rb

```diff
@@ -37,49 +37,29 @@
     end
   end
 
-  describe 'setup links to redirect' do
+  describe 'setup single url redirect' do
     before do
       node.set['redirector']['urls'] = {
         'john-doe.example.org' => {
-          '/foo' => 'http://example.org/bar',
-          '/' => 'http://example.com',
-          '/bar' => 'http://example.com/bar'
+          '/' => 'http://example.com'
```

Comment:
This test should actually fail, but it doesn't. Looking at it, the old tests should also have failed, because the protocol `http://` is hard-coded into the template. I wonder how to implement the test in a better way?